websites/D/Desu-Online/presence.ts

```diff
@@ -0,0 +1,166 @@
+						.substring(2, searchvalue.length)
+						.replaceAll("+", " ")
+						.replaceAll("%20", " ");
+				} else if (searchvalue.startsWith("?s=")) {
+					presenceData.details = "Wyszukuje po frazie:";
+					presenceData.state = searchvalue
+						.substring(3, searchvalue.length)
+						.replaceAll("+", " ")
+						.replaceAll("%20", " ");
+				}
+			}
+		} else if (pth === "/") {
+			presenceData.details = "Przegląda stronę główną...";
+			presenceData.smallImageKey =
+				"https://cdn-icons-png.flaticon.com/512/5974/5974636.png";
+		} else if (pth.startsWith("/anime/")) {
+			if (pth === "/anime/") presenceData.details = "Przegląda serie...";
+			else if (pth === "/anime/list-mode/")
+				presenceData.details = "Przegląda listę serii...";
+			else {
+				presenceData.details = "Przegląda serię:";
+				presenceData.state = document.querySelectorAll("h1")[0].textContent;
+				presenceData.largeImageKey = document
+					.querySelector("img.ts-post-image")
+					.getAttribute("src");
+				presenceData.buttons = [
+					{
+						label: "Odwiedź",
+						url: document.URL,
+					},
+				];
+			}
+		} else if (pth === "/harmonogram/")
+			presenceData.details = "Przegląda harmonogram...";
+		else if (pth === "/zakladki/")
+			presenceData.details = "Przegląda swoje zakładki...";
+		else if (pth.startsWith("/blog/")) {
+			if (pth === "/blog/") {
+				presenceData.details = "Przegląda bloga:";
+				presenceData.state = "Strona 1";
+			} else if (pth.startsWith("/blog/page/")) {
+				presenceData.details = "Przegląda bloga:";
+				presenceData.state = `Strona ${
+					document.location.pathname.split("/")[3]
+				}`;
+			} else {
+				presenceData.details = "Czyta bloga:";
+				presenceData.state = document
+					.querySelectorAll("h1")[0]
+					.textContent.replace("\n", "");
+				presenceData.buttons = [{ label: "Czytaj", url: document.URL }];
+			}
+		} else if (pth === "/gatunki/")
+			presenceData.details = "Przegląda gatunki...";
+		else if (pth.startsWith("/season/")) {
+			presenceData.details = "Przegląda sezon:";
+			presenceData.state = document.querySelectorAll("h1")[0].textContent;
+		} else if (pth === "/sezony/") presenceData.details = "Przegląda sezony...";
+		else if (pth.startsWith("/label/")) {
+			if (pth === "/label/news/") presenceData.details = "Przegląda newsy...";
+			else if (pth === "/label/recenzja/")
+				presenceData.details = "Przegląda recenzje...";
+		} else if (pth === "/ekipa-portalu/")
+			presenceData.details = "Przegląda ekipę portalu...";
+		else if (pth === "/polityka-prywatnosci/")
+			presenceData.details = "Przegląda politykę prywatności...";
+		else if (pth.startsWith("/genres/")) {
+			presenceData.details = "Przegląda gatunek:";
+			presenceData.state = `${document.querySelectorAll("h1")[0].textContent}`;
+			if (pth.includes("page")) {
+				presenceData.state = `${
+					document.querySelectorAll("h1")[0].textContent
+				} (Strona ${document.location.pathname.split("/")[4]})`;
```

Comment:
You already have a variable for `document.location.pathname`.
```suggestion
				} (Strona ${pth.split("/")[4]})`;
```